src/tests/functional/shared_test_classes/src/single_layer/comparison.cpp

```diff
@@ -78,24 +78,39 @@ void ComparisonLayerTest::SetUp() {
 }
 
 InferenceEngine::Blob::Ptr ComparisonLayerTest::GenerateInput(const InferenceEngine::InputInfo &inputInfo) const {
-    auto blob = LayerTestsUtils::LayerTestsCommon::GenerateInput(inputInfo);
+    InferenceEngine::Blob::Ptr blob;
 
     if (comparisonOpType == ComparisonTypes::IS_FINITE || comparisonOpType == ComparisonTypes::IS_NAN) {
```

Comment:
As I can see, at the moment, `jit_emitter` for `IsInf` always works only for case `detectNegative = true, detectPositive = true` so I have obvious question: why CI is green? :smile: And if I understand correctly, at the moment we generate specific inputs only for `IsFinite` and `IsNan`. What's about `IsInf`?

Reply:
There is another test for that https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/is_inf.cpp